ISA-XLSX.md

```diff
@@ -0,0 +1,741 @@
+
+This section MUST contain the following labels, with the specified datatypes for values supported:
+
+| Label                   | Datatype                  | Description                                                                                                                                                                     |
+|-------------------------|---------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Term Source Name        | String                    | The name of the source of a term; i.e. the source controlled vocabulary or ontology. These names will be used in all corresponding Term Source REF fields that occur elsewhere. |
+| Term Source File        | String (file name or URI) | A file name or a URI of an official resource.                                                                                                                                   |
+| Term Source Version     | String                    | The version number of the Term Source to support terms tracking.                                                                                                                |
+| Term Source Description | String                    | Use for disambiguating resources when homologous prefixes have been used.                                                                                                       |
+
+**Example**
+
+For example, the `ONTOLOGY SOURCE REFERENCE` section of an ISA-XLSX `isa.investigation.xlsx` file may look as follows:
+
+|                    |       |       |       |             |      |
+|--------------------|-------|-------|-------|-------------|------|
+| ONTOLOGY SOURCE REFERENCE | 
+| Term Source Name  | CHEBI | EFO | OBI | NCBITAXON | PATO |
+| Term Source File  | [http://data.bioontology.org/ontologies/CHEBI](http://data.bioontology.org/ontologies/CHEBI) | [http://data.bioontology.org/ontologies/EFO](http://data.bioontology.org/ontologies/EFO) | [http://data.bioontology.org/ontologies/OBI](http://data.bioontology.org/ontologies/OBI) | [http://data.bioontology.org/ontologies/NCBITAXON](http://data.bioontology.org/ontologies/NCBITAXON) | [http://data.bioontology.org/ontologies/PATO](http://data.bioontology.org/ontologies/PATO) |
+| Term Source Version | 78  | 111 | 21  | 2         | 160 |
+| Term Source Description | Chemical Entities of Biological Interest Ontology | Experimental Factor Ontology | Ontology for Biomedical Investigations | National Center for Biotechnology Information (NCBI) Organismal Classification | Phenotypic Quality Ontology |
+
+
+## INVESTIGATION section
+
+This section is organized in several subsections, described in detail below. The Investigation section provides a
+flexible mechanism for grouping two or more Study files where required. When only one Study is created, the values in
```

Comment:
>  When only one Study is created, the values in
this section SHOULD be left empty and the relevant metadata values recorded in the Study section only.

Why?